### PR TITLE
Added missing favicon

### LINKED ISSUE #202: [OTHER] Missing favicon
### What would you like to share?

There is no favicon for the complete website.
I can add a handshake logo as a favicon.
Annotation 2022-03-18 022038

### PR DESCRIPTION
**Pull request by mr-palindrome**
## 🛠️ Fixes Issue
[OTHER] Missing favicon 

Closes #202 

## 👨‍💻 Changes proposed

Added missing favicon in the base HTML and login page using static files by creating a new URL for favicons.
## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
added favicon
CodingEasy

### CODE REVIEW

File: CodingEasy/urls.py [hunk 1]
```diff
@@ -17,7 +17,15 @@
 from django.contrib import admin
 from django.urls import path, include
 
+#for favison
```
Comment (singhmansi25) [inline CodingEasy/urls.py:20]: Please correct this comment to #for favicon, to avoid any confusion for future.

Comment (singhmansi25) [reply CodingEasy/urls.py:20]: And update this comment in all the files, where its written.

Comment (mr-palindrome) [reply CodingEasy/urls.py:20]: yeah sure

**Comment by arpit456jain** [comment]
@singhmansi25 @raghavTinker

**Review by KKhushhalR2405 — APPROVED** [review]
LGTM @arpit456jain

**Pull request merged**